Character.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+
+class Character
+{
+private:
+	std::string name;
+	unsigned int HP, DMG;
+public:
+	Character(std::string characterName, int hitPoints, int damagePoints);
+	void setHP(int hitPoints);
+	void setDMG(int damagePoints);
+	int getHP();
+	int getDMG();
+	void setName(std::string characterName);
+	std::string getName();
```

Comment:
Ahol lehet, ott használjunk const adattagokat és metódusokat is. Setter-ek használata nélkül kell megoldani ezt a feladatot.

Reply:
@vizvezetek A settereket hogyan mellőzzük?  

1. Public-ba kerüljenek a változtatandó értékek
2. Készítsünk az osztályon belül függvényt a támadásra
3. Hozzon létre a main minden körben újabb példányt az előzőből másolt, valamint az újonnan számolt értékekkel
4. Egyéb

Reply:
1. Ha Publicba kerül egy adattag, akkor azzal felborítod az OO alapelveket. Természetesen van helyzet, amikor ezt a módszert kell allkalmazni, de jelenleg nem.
2. Kiváló ötlet!
3. Az kicsit túbonyolított, és nem túl elegáns megoldás.
4. -